backend/src/src-users/src/main/java/com/siemens/sw360/users/UserHandler.java

```diff
@@ -85,4 +85,10 @@ public RequestStatus updateUser(User user) throws TException {
         return db.updateUser(user);
     }
 
+    public RequestStatus sendMailForAcceptedModerationRequest(String userEmail) throws TException {
+        assertNotNull(userEmail);
```

Comment:
You could add a function `assertValidEmail(...)` but that is a whole other problem. See
* http://ex-parrot.com/~pdw/Mail-RFC822-Address.html or
* http://stackoverflow.com/questions/201323/using-a-regular-expression-to-validate-an-email-address